script/openqa-clone-custom-git-refspec

```diff
@@ -82,18 +82,30 @@ clone_job() {
     local casedir="${casedir:-"$repo#$branch"}"
     local GROUP="${GROUP:-0}"
     local dry_run="${dry_run:-""}"
-    local cmd="$dry_run openqa-clone-job --skip-chained-deps --within-instance \"$host\" \"$job\" _GROUP=\"$GROUP\" TEST=\"$test\" BUILD=\"$build\" CASEDIR=\"$casedir\" PRODUCTDIR=\"$productdir\" NEEDLES_DIR=\"$needles_dir\" $args"
+    cmd=(
+        $dry_run openqa-clone-job --skip-chained-deps --within-instance
+        "$host" "$job"
+        _GROUP="$GROUP"
+        TEST="$test"
+        BUILD="$build"
+        CASEDIR="$casedir"
+        PRODUCTDIR="$productdir"
+        NEEDLES_DIR="$needles_dir"
+        ${args[*]}
+    )
+    $debug && echo "cmd: ${cmd[@]}"
```

Comment:
that would be the same as "dry_run", wouldn't it?

Reply:
yeah, similar. I can take it out

Reply:
How should `debug` and `dry_run` differ? Just the `cmd: ` prefix?

Neither is documented... perhaps adding docs would be the first step, the second would be seeing what's worth adding?